components/Discussion/Comments.js

```diff
@@ -96,8 +96,7 @@ const Comments = props => {
   } = props
 
   const router = useRouter()
-  const discussionHref = getDiscussionHref(discussion)
-
+  const discussionHref = getDiscussionUrlObject(discussion)
```

Comment:
Since you went through the trouble to rename `getDiscussionHref` to `getDiscussionUrlObject`, rename `discussionHref`, too?

Reply:
done